mlflow/ml-package-versions.yml

```diff
@@ -331,7 +333,11 @@ spark:
         cat $SAGEMAKER_OUT;
         exit 1
       fi
-      pytest tests/spark --ignore tests/spark/autologging
+      pytest tests/spark --ignore tests/spark/autologging --ignore tests/spark/test_spark_connect_model_export.py
+
+      if [[ $(python -c "from packaging.version import Version;import pyspark;print(Version(pyspark.__version__) >= Version('3.5'))") = "True" ]]; then
+        pytest tests/spark/test_spark_connect_model_export.py;
+      fi
```

Comment:
I think we can add `pytest.mark.skipif` in `tests/spark/test_spark_connect_model_export.py`.

Reply:
There is an issue, if we skip all test in `pytest tests/spark/test_spark_connect_model_export.py;`, pytest process exits with code 5, this breaks CI.

Reply:
Does `pytest tests/spark --ignore tests/spark/autologging` exit with non-zero?